doc/introduction/operations.rst

```diff
@@ -60,6 +60,7 @@ Operator to Operator functions
     ~pennylane.map_wires
     ~pennylane.dot
     ~pennylane.evolve
+    ~pennylane.iterative_qpe
```

Comment:
Not blocking, just a suggestion for a future doc-string PR. This returns a measurement process (MP). Since we are starting to add more support for functions which return MP (qml.measure, qml.cond), we should add a new section. Something like `Operator to Measurement`

Reply:
It makes sense 👍
I would say that as soon as we have 2 of this type, we can make the section

Reply:
Or are you saying here that qml.measure and qml.cond is in the same family as iterative_qpe?

Reply:
I am saying that `qml.measure` and `qml.cond` are in the same family, and we can make a new section now. But for now, I would say not to do it in this PR. But rather have a follow up PR to update docs. This way its not blocking the feature and we have more time to discuss with the product team.